schwab/streaming.py

```diff
@@ -1041,113 +1094,128 @@ class LevelOneFuturesFields(_BaseFieldEnum):
         streaming-data#_Toc504640603>`__
         '''
 
-        #: UNKNOWN
-        FIELD_0 = 0
+        #: Ticker symbol in upper case.
+        SYMBOL = 0
 
-        #: UNKNOWN
-        FIELD_1 = 1
+        #: Current Best Bid Price
+        BID_PRICE = 1
 
-        #: UNKNOWN
-        FIELD_2 = 2
+        #: Current Best Ask Price
+        ASK_PRICE = 2
 
-        #: UNKNOWN
-        FIELD_3 = 3
+        #: Price at which the last trade was matched
+        LAST_PRICE = 3
 
-        #: UNKNOWN
-        FIELD_4 = 4
+        #: Number of contracts for bid
+        BID_SIZE = 4
 
-        #: UNKNOWN
-        FIELD_5 = 5
+        #: Number of contracts for ask
+        ASK_SIZE = 5
 
-        #: UNKNOWN
-        FIELD_6 = 6
+        #: Exchange with the best bid
+        BID_ID = 6
 
-        #: UNKNOWN
-        FIELD_7 = 7
+        #: Exchange with the best ask
+        ASK_ID = 7
 
-        #: UNKNOWN
-        FIELD_8 = 8
+        #: Aggregated contracts traded throughout the day, including pre/post market hours.
+        TOTAL_VOLUME = 8
 
-        #: UNKNOWN
-        FIELD_9 = 9
+        #: Number of contracts traded with last trade
+        LAST_SIZE = 9
 
-        #: UNKNOWN
-        FIELD_10 = 10
+        #: Time of the last quote in milliseconds since epoch
+        QUOTE_TIME_MILLIS = 10
 
-        #: UNKNOWN
-        FIELD_11 = 11
+        #: Time of the last trade in milliseconds since epoch
+        TRADE_TIME_MILLIS = 11
 
-        #: UNKNOWN
-        FIELD_12 = 12
+        #: Day's high trade price
+        HIGH_PRICE = 12
 
-        #: UNKNOWN
-        FIELD_13 = 13
+        #: Day's low trade price
+        LOW_PRICE = 13
 
-        #: UNKNOWN
-        FIELD_14 = 14
+        #: Previous day's closing price
+        CLOSE_PRICE = 14
+
+        #: Primary "listing" Exchange
+        EXCHANGE_ID = 15
+
+        #: Description of the product
+        DESCRIPTION = 16
+
+        #: Exchange where last trade was executed
+        LAST_ID = 17
+
+        #: Day's Open Price
+        OPEN_PRICE = 18
+
+        #: Current Last-Prev Close
+        NET_CHANGE = 19
 
-        #: UNKNOWN
-        FIELD_15 = 15
+        #: Current percent change
+        FUTURE_CHANGE_PERCENT = 20
```

Comment:
I believe this may have been done accidentally since other fields were modified where it _does_ make sense to have the `FUTURE` prefix.  Would it make more sense to keep this consistent with other contract/ticker types? 

`CHANGE_PERCENT` instead of `FUTURE_CHANGE_PERCENT`